models/comment.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model } = require('mongoose')
+const { FIELD_CANNOT_BE_EMPTY } = require('~/consts/errors')
+const { COOPERATION, COMMENT, USER } = require('~/consts/models')
+
+const commentSchema = new Schema(
+  {
+    text: {
+      type:String,

```

Comment:
set min and max length

Reply:
i have added min length but i am not sure that we have two set max length cause if tutor will need to stay in comment some a lot of info he will need to divide it on several comments